plugins/catalog-backend-module-bitbucket/src/index.ts

```diff
@@ -14,5 +14,10 @@
  * limitations under the License.
  */
 
+/**
+ * A Backstage catalog backend module that helps integrate towards GitLab
```

Comment:
integrate towards **Bitbucket**

Reply:
Haha! I had accidentally added `index.ts` to the exclusions in Find in files. :) Fixing